### PR TITLE
Aao/leet code practice

### PR DESCRIPTION
**Pull request by marks214**
please check it out!

### CODE REVIEW

**Comment by cescarez** [comment]
:birthday:

**Comment by RachaelGomez** [comment]
dang real code nice

File: green-color.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+green
```
Comment (cescarez) [inline green-color.txt:1]: but what shade of green?

**Review by RachaelGomez — APPROVED** [review]
very green!

File: green-color.txt (hunk 1 above)
Comment (marks214) [reply green-color.txt:1]: all the shades, all together

**Pull request merged**